tests/cases/compiler/lateBoundDestructuringImplicitAnyError.ts

```diff
@@ -0,0 +1,18 @@
+// @noImplicitAny: true
+let named = "foo";
+let {[named]: prop} = {prop: "foo"};

```

Comment:
```ts
const named = Symbol("");
let { [named]: prop } = strIndexed;
void prop;``
```

today that is `string` ... we probably wanna keep it this way until we have a symbol indexer.